WebContent/WOH-addRestaurant.jsp

```diff
@@ -53,20 +53,20 @@
 						<div class="form-group">
 						<label class="control-label col-sm-2" for="WOH-addRestaurant-distanceInput">Distance</label>
 							<div class="col-sm-10">
-	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" name = "distance">
+	         					<input type = "text" class="form-control WOH-addRestaurant-distanceInput" value = 0 name = "distance">
```

Comment:
change this to input type range.
<input type="range" name="distance" min="2" max="200">
http://www.eyecon.ro/bootstrap-slider/
this website has some cool documentation for how to dress it up.